src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -91,7 +93,7 @@
     private static String hg19Chr19Ref;
 
     static {
-        if (!doDebugTests) {
+        if (!enableFullScaleTests ) {
             tmpOutDir = createTempDir("funcotatorTmpFolder");
         } else {
             tmpOutDir = new File("funcotatorTmpFolder" + File.separator);
```

Comment:
If you intentionally don't want this deleted, add a comment to that effect.

Reply:
Fixed.